packages/repository/src/common-types.ts

```diff
@@ -98,5 +98,6 @@ export interface Count {
  */
 export const CountSchema = {
   type: 'object',
+  title: 'Count',
```

Comment:
Do we want to make the title qualified such as `loopback.Count`? Would it confuse code generation tools?

Reply:
You are right, I was concerned by naming collisions that tend to cause issues with some generation tools in particular circumstances.  Qualifying this property will result in a string conversion such as camelCase/PascalCase, so no problem(on aforementioned tool).

Updated to ``loopback.count`` as per feedback.